### PR TITLE
LLM: Rename low bit layer

### PR DESCRIPTION
**Pull request by leonardozcm**
## Description

Rename LinearQuant to LinearLowBit layer.

### CODE REVIEW

File: python/llm/src/bigdl/llm/transformers/linear_quant.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def ggml_matmul_src1_x_src0_t(src0: torch.Tensor,
     return result_t
 
 
-class LinearQuant(nn.Linear):
+class LinearLowBit(nn.Linear):
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/transformers/linear_quant.py:214]: change to `LowBitLinear`

File: python/llm/src/bigdl/llm/transformers/convert.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
 
 def _replace_with_quant_linear(model, qtype, modules_to_not_convert=None,
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/transformers/convert.py:46]: change to `_replace_with_low_bit_linear`

File: python/llm/src/bigdl/llm/transformers/linear_quant.py (hunk 1 above)
Comment (jason-dai) [reply python/llm/src/bigdl/llm/transformers/linear_quant.py:214]: and change the file name to `low_bit_linear.py`

**Comment by jason-dai** [comment]
LGTM

**Review by hoshibara — APPROVED** [review]

**Pull request merged**